lib/File/Copy.pm

```diff
@@ -24,7 +22,7 @@ sub syscopy;
 sub cp;
 sub mv;
 
-$VERSION = '2.37';
+$VERSION = '2.38';
 
 require Exporter;
 @ISA = qw(Exporter);
```

Comment:
Indirect import method call here got broken by the import of the 5.035007 feature bundle 😄

Reply:
Fixed in fbf19e9285